### DISCUSSION TITLE
Renovate considers a merged PR "unmerged". Maybe due to main branch not having the bump anymore?

### DISCUSSION THREAD

**Discussion by Andre601** [renovatebot/renovate#27419] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.198.3

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### What is your question?

I just received the following notification on a merged Pull request:
> ### Renovate Ignore Notification
> 
> Because you closed this PR without merging, Renovate will ignore this update (`v3.2.0`). You will get a PR once a newer version is released. To ignore this dependency forever, add it to the `ignoreDeps` array of your Renovate config.
> 
> If you accidentally closed this PR, or if you changed your mind: rename this PR to get a fresh replacement PR.

This is confusing since the PR was merged, as I mentioned. Does Renovate only check the target branch for if the bumped version is included, to determine a PR merged? If so, why that and not just check a PRs merged status?

This isn't a real issue for me. Just a question that is on my mind and that I hope to get answers for.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Renovate doesn't look on main branch - it checks whether the platform returns the status as closed-unmerged or closed-merged. So if you see that message then it thinks it's closed-unmerged
